docs/source/user-guide/configuration/configuration-settings.rst

```diff
@@ -0,0 +1,807 @@
+
+.. _use-only-tar-bz2:
+
+Force conda to download only .tar.bz2 packages (use_only_tar_bz2)
+-----------------------------------------------------------------
+
+Conda 4.7 introduced a new ``.conda`` package file format.
+``.conda`` is a more compact and faster alternative to ``.tar.bz2`` packages.
+It's thus the preferred file format to use where available.
+
+Nevertheless, it's possible to force conda to only download ``.tar.bz2`` packages
+by setting the ``use_only_tar_bz2`` boolean to ``True``.
+
+The default is ``False``.
+
+**Example:**
+
+.. code-block:: yaml
+
+  use_only_tar_bz2: True
+
+.. note::
+
+   This is forced to ``True`` if conda-build is installed and older than 3.18.3,
+   because older versions of conda break when conda feeds it the new file format.
+
+Conda-build configuration
+=========================
+
+.. _specify-root-dir:
+
+Specify conda-build output root directory (root-dir)
+----------------------------------------------------
+
+Build output root directory. You can also set this with the
+``CONDA_BLD_PATH`` environment variable. The default is
+``<CONDA_PREFIX>/conda-bld/``. If you do not have write
+permissions to ``<CONDA_PREFIX>/conda-bld/``, the default is
+``~/conda-bld/``.
+
+**Example:**
+
+.. code-block:: yaml
+
+  conda-build:
+      root-dir: ~/conda-builds
+.. _specify-output-folder:
+
+Specify conda-build build folder (conda-build 3.16.3+) (output_folder)
+----------------------------------------------------------------------
+
+Folder to dump output package to. Packages are moved here if build or test
+succeeds. If unset, the output folder corresponds to the same directory as
+the root build directory (``root-dir``).
+
+.. code-block:: yaml
+
+   conda-build:
+       output_folder: conda-bld
+
+.. _pkg_format:
+
+Specify conda-build package version (pkg_version)
+-------------------------------------------------
+
+Conda package version to create. Use ``2`` for ``.conda`` packages. If not set, conda-build defaults to ``.tar.bz2``.
+
+.. code-block:: yaml
+
+   conda-build:
+      pkg_format: 2
+
+.. _auto-upload:
+
+Automatically upload conda-build packages to Anaconda.org (anaconda_upload)
+---------------------------------------------------------------------------
+
+Automatically upload packages built with conda-build to
+`Anaconda.org <http://anaconda.org>`_. The default is ``False``.
+
+**Example:**
+
+.. code-block:: yaml
+
+  anaconda_upload: True
+
+.. _anaconda-token:
+
+Token to be used for Anaconda.org uploads (conda-build 3.0+) (anaconda_token)
+-----------------------------------------------------------------------------
+
+Tokens are a means of authenticating with Anaconda.org without logging in.
+You can pass your token to conda-build with this ``.condarc`` setting, or with a CLI
+argument. This is unset by default. Setting it implicitly enables
+``anaconda_upload``.
+
+  .. code-block:: yaml
+
+     conda-build:
+         anaconda_token: gobbledygook
+
+.. _quiet:
```

Comment:
```suggestion
.. code-block:: yaml

  conda-build:
    anaconda_token: gobbledygook

.. _quiet:
```

Reply:
The original spacing here makes the code block appear like it's a code block within a code block (see the screenshot below):

<img width="759" alt="Screenshot 2024-01-08 at 1 28 42 PM" src="https://github.com/conda/conda/assets/28930622/431530f8-ae50-4426-af4f-304c066c992f">